test/test_p4est_2d.jl

```diff
@@ -496,10 +496,10 @@ end
     @test_trixi_include(joinpath(EXAMPLES_DIR,
                                  "elixir_euler_supersonic_cylinder_sc_subcell.jl"),
                         l2=[
-                            0.11085870166618325,
-                            0.23309905989870722,
+                            0.11085869100230632,
+                            0.23309900854310292,
                             0.13505351590735631,
-                            0.7932047512585592
+                            0.7932046814135086
```

Comment:
If we already know that this stuff is prone to changes, should we instead update the test tolerances?

Reply:
Hm good question. It would definitely make things easier in the future.
On the other hand, we would have to set the tolerances to something like `1e-6`. I'm not sure if that's a good idea. Especially, since the resulting errors were always identical using the same setup. So, the CI runs were always a good check whether everything stayed the same.

But of course, we probably would have to reset the errors for every new Julia version. And locally, some tests might fail all the time (you get used to it :sweat_smile: )

Reply:
I would prefer keeping the low tolerance in order to not miss some unwanted changes. Updating the error of a couple of tests every new Julia version feels like no big deal to me.

Reply:
Ok, thanks. I am not sure whether we will switch our CI system completely to Julia v1.11 - but I will merge this PR for now. Thanks